.binder/environment.yml

```diff
@@ -8,7 +8,7 @@ dependencies:
 - pyiron_base
 - lammps
 - iprpy-data
-- gpaw
+#- gpaw
```

Comment:
Is there a reason why `gpaw` is removed?

Reply:
when I tested it with [the equivalent gitlab repository](https://gitlab.mpcdf.mpg.de/pyiron/collaborative_pyiron_in_binder/-/tree/main/) there was an error and the auto completion did not work. (Not even when I tried to use auto completion alone, so without working with more people on one notebook.)